src/components/Calculator.js

```diff
@@ -0,0 +1,25 @@
+import { Component } from 'react';
+import '../index.css';
+import Row from './Row';
+
```

Comment:
- [ ] Your _App_ component _(App.js)_ is not classed based. Kindly change this component to a class-based component.

Reply:
thank you for your comment and the problem was solved